### ISSUE TITLE
Numbers and commas spoken in Polish

### ISSUE THREAD

**Issue by georgjaehnig** [martinrue/vocx#2]
"3,4 milionoj" is spoken as "trzy przecinek cztery".

Probably a number preprocessor is needed, writing numbers as their Esperanto words.

**Comment by martinrue** [comment]
Good call, I'll think about the best way to do this.

**Comment by martinrue** [comment]
I've just deployed a change, adding the ability to say numbers. 🙌

Examples: 
`12345` should now be spoken: `dekdu mil, tricent kvardek kvin`. 
`49,7` should now be spoken: `kvardek naŭ punkto sep`.

It won't process very large numbers, but it should handle common cases up to hundreds of millions.

(Dates/times are a separate issue that still needs to be fixed.)

**Comment by georgjaehnig** [comment]
Dziękuję! Seems to work well.

But why `49,7` with `punkto`, not `komo`? In Esperanto, apparently [komo is used](https://esperanto.stackexchange.com/questions/1463/kio-estas-la-esperanta-nombro-disigilo-komo-a%C5%AD-mallar%C4%9Da-spaceto).

**Comment by georgjaehnig** [comment]
Ho, ŝajne ĉe `24%` ĝi daure estas pola elparolo.

**Comment by martinrue** [comment]
> Dziękuję! Seems to work well.
> 
> But why `49,7` with `punkto`, not `komo`? In Esperanto, apparently [komo is used](https://esperanto.stackexchange.com/questions/1463/kio-estas-la-esperanta-nombro-disigilo-komo-a%C5%AD-mallar%C4%9Da-spaceto).

Oops – I'll change it to say `komo` instead... thanks :)

**Comment by martinrue** [comment]
> Ho, ŝajne ĉe `24%` ĝi daure estas pola elparolo.

Yes – currently it only looks for numbers. Anything that doesn't look like a number by itself will be ignored.
